buildah_intro/buildah_intro.sh

```diff
@@ -264,17 +312,19 @@ buildah_from_dockerfile() {
 
 clean_images_and_containers() {
 
-    read -p "buildah rm -a"
-    buildah rm -a
+    read -p "sudo buildah rm -a"
+    sudo buildah rm -a
     echo
-    read -p "buildah rmi -a -f"
-    buildah rmi -a -f
+    read -p "sudo buildah rmi -a -f"
+    sudo buildah rmi -a -f
 
     echo
     read -p "Enter to continue"
     clear
 }
 
+buildah_from_dockerfile_rootless
```

Comment:
I think you want to remove this line, you have it below.  I think you're running it there intentionally after clearing the containers/images.  Do you want to clear them again after this runs?

Reply:
Yes, that was for testing,  Removed.